### DISCUSSION TITLE
How To Message A Contributor/s

### DISCUSSION THREAD

**Discussion by MysticMerlin** [community/community#61947] [New to GitHub]
I can't find/see anywhere else that I can ask a simple question which may have a simple answer.
I hope I'm not to break any laws on here by asking the following.
There are problems with the youtube-dl/yt-dlp software & it is being worked on & like a lot of people I'm waiting for a fix.
Obviusly, until a fix has been done I can't do anything & so I just wanted to get an update on how things are going.
The two people I'd like to message are dirkf & Vangelis66 but I can't see how to message them.
So, could anybody enlighten me how to do that please?
Thanks in advance.

**Comment by queenofcorgis** [comment]
GitHub does not offer private messaging, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by MysticMerlin** [reply]
Thanks for your quick reply queenofcorgis. I will try the above later on today & I didn't get chance to check for replies last night in the end.

**Reply by queenofcorgis** [reply]
Happy to help - hope you get that update 🚀
